### ISSUE TITLE
Old pegged versions removed for CentOS

### ISSUE THREAD

**Issue by brycefisher** [fluent/fluent-bit#549]
Since the release 0.12.15, I'm unable to install 0.12.14. Is the removal of an old version intentional?

**Comment by edsiper** [comment]
hi @brycefisher 

older versions are only deleted from the Yum repository index but the files remains available, to download 0.12.14 you can use the following link:

http://packages.fluentbit.io/centos/7/td-agent-bit-0.12.14-1.x86_64.rpm

**Comment by brycefisher** [comment]
Gotcha! So, if I want to peg my version of fluentbit, I should hardcode the url directly instead of relying on a Yum repository index version.

**Comment by edsiper** [comment]
Yes, but I strongly recommend you use the latest 0.12 stable instead of an older version.
